components/BlogSection.tsx

```diff
@@ -0,0 +1,125 @@
+import BlogPost from './BlogPost';
+import styles from './BlogSection.module.css';
+
+type BlogSectionProps = {
+  title?: string;
+  variation: string;
+};
+
+const POST_DUMMY_DATA = {
+  title: 'My first post',
+  category: 'Category',
+  summary: 'This is the first post published in this blog.',
+  datePublished: '2021-08-31',
+  featuredImageUrl:
+    'https://images.unsplash.com/photo-1630359486912-4b085e38f1e7?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=748&q=80',
+  author: 'Juan Pérez',
+};
+
+const BlogSection = ({ title, variation }: BlogSectionProps) => {
+  console.log(variation);
+  return (
+    <div className={`${styles['blog-section']} ${styles[`blog-section--${variation}`]}`}>
```

Comment:
We discussed this on our call earlier, but see how this JSX is very long and nested.

The best we should look after when writing functions in JS (This applies to React components being a function) is to keep them small and doing one thing only.

A pattern we should try to stick to as much as possible.